lib/openfoodfacts.dart

```diff
@@ -499,6 +501,17 @@ class OpenFoodAPIClient {
         user: user, queryType: queryType);
   }
 
+  static Future<Map<String, TaxonomyPackaging>?> getTaxonomyPackagings(
+    final TaxonomyPackagingQueryConfiguration configuration, {
+    final User? user,
+    final QueryType? queryType,
+  }) =>
+      getTaxonomy<TaxonomyPackaging, TaxonomyPackagingField>(
+        configuration,
+        user: user,
```

Comment:
Do we even need to put a user here

Reply:
I don't know, I've just copied the code from other taxonomies.